pkg/operator/ceph/controller/controller_utils.go

```diff
@@ -138,22 +139,30 @@ func EnforceHostNetwork() bool {
 }
 
 func SetRevisionHistoryLimit(data map[string]string) {
-	strval := k8sutil.GetValue(data, revisionHistoryLimitSettingName, revisionHistoryLimitDefaultValue)
-	if strval != "" {
+	strval := k8sutil.GetValue(data, revisionHistoryLimitSettingName, revisionHistoryLimitDefaultStr)
+	var numval int32
+	var limit int32 
+	 _ = limit
```

Comment:
What is this line for? you can remove it

Reply:
@travisn  wrote:
> What is this line for?

It was an  earkier attempt to fix an declared but unused vet/build error.

> you can remove it

It is gone now.